webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
```

Comment:
Missing copyright header? (Do we need that on every file? There are some other config files that don't have it.)